py/objgenerator.c

```diff
@@ -252,7 +252,18 @@ STATIC MP_DEFINE_CONST_FUN_OBJ_2(gen_instance_send_obj, gen_instance_send);
 
 STATIC mp_obj_t gen_instance_close(mp_obj_t self_in);
 STATIC mp_obj_t gen_instance_throw(size_t n_args, const mp_obj_t *args) {
-    mp_obj_t exc = (n_args == 2) ? args[1] : args[2];
+    // Python allows the following variations on the arguments:
+    //  1. args[1]=exception instance; args[2]=None
```

Comment:
I didn't say +1 to this, because I wanted to check this matter myself, because https://docs.python.org/3/reference/expressions.html#generator.throw is very-very bare. Taken literally, it doesn't even say that there can be single argument which is an exception instance. I'd also say this comment is confusing, because it kinda says there should be 2 arguments to .throw().